bridge/opentracing/bridge.go

```diff
@@ -620,6 +620,25 @@ func (s fakeSpan) SpanContext() trace.SpanContext {
 	return s.sc
 }
 
+// adapt OpenTracing TextMapWriter to OpenTelemetry propagation.TextMapCarrier
+type textMapWriterAdapter struct {
+	t ot.TextMapWriter
+}
```

Comment:
```suggestion
}

var _ propagation.TextMapCarrier = textMapWriterAdapter{}
var _ propagation.TextMapCarrier = textMapReaderAdapter{}
```

I'd like to see tests, but at the least we should assert that the interfaces are satisfied.

Reply:
The effect of these assertions is already covered by the assignments to `tmCarrier` which is of type `propagation.TextMapCarrier`.